app/src/pages/prompt/PromptInvocationParameters.tsx

```diff
@@ -9,7 +9,7 @@ import { safelyStringifyJSON } from "@phoenix/utils/jsonUtils";
 
 import { PromptInvocationParameters__main$key } from "./__generated__/PromptInvocationParameters__main.graphql";
 
-function PromptInvocationParameterItem({
+export function PromptInvocationParameterItem({
```

Comment:
unnecessary export?

Reply:
Ah yeah, good catch. I initially exported and reused this component but ultimately decided against it